src/components/common/Tab.js

```diff
@@ -1,10 +1,12 @@
-import React, { Component } from 'react';
+import React, { Component, useContext } from 'react';
 import PropTypes from 'prop-types';
+import { Badge } from 'antd';
 class Tab extends Component {
```

Comment:
Most of this repos code is done using React Hooks. I'm curious about the use of Class components here?

Reply:
Oh, we had actually considered using a class component in this case because...